source/_docs/installation/raspberry-pi.markdown

```diff
@@ -63,7 +63,7 @@ Next up is to create and change to a virtual environment for Home Assistant Core
 ```bash
 sudo -u homeassistant -H -s
 cd /srv/homeassistant
-python3.8 -m venv .
+python3 -m venv .
```

Comment:
Doesn't look like a typo to me. `python3` usually points to the latest Python 3.x release that is available on the system. If there is an older Python release available like Python 3.2 `python3` would invoke that and this would lead to an error.

Using `python3.8` ensure that the correct release is used.